parquet-column/src/main/java/org/apache/parquet/column/page/PageReader.java

```diff
@@ -37,4 +39,9 @@ public interface PageReader {
    * @return the next page in that chunk or null if after the last page
    */
   DataPage readPage();
+
+  /**
+   * Close the page reader. By default it is no-op.
+   */
+  default void close() throws IOException {}
```

Comment:
Should we add `Closeable` as an implemented interface?

Reply:
Cool. I know a lot of times I check if instances of closeable and then call close.